app/DoctrineMigrations/Version20231117160815.php

```diff
@@ -32,9 +32,6 @@ public function down(Schema $schema): void
         $this->addSql('ALTER TABLE hub DROP CONSTRAINT FK_4871CE4D5BC85711');
         $this->addSql('DROP INDEX UNIQ_4871CE4D5BC85711');
         $this->addSql('ALTER TABLE hub DROP business_account_id');
-        $this->addSql('ALTER TABLE restaurant ADD business_account_id INT DEFAULT NULL');
-        $this->addSql('ALTER TABLE restaurant ADD CONSTRAINT fk_eb95123f5bc85711 FOREIGN KEY (business_account_id) REFERENCES business_account (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
-        $this->addSql('CREATE INDEX idx_eb95123f5bc85711 ON restaurant (business_account_id)');
```

Comment:
This migration should/may already have been executed, better not modify it, or migration `Version20231219182856` will not work, because it will try to drop this column.

Reply:
Sorry, I didn't notice those changes were in the `down` method.